core/src/com/unciv/logic/civilization/diplomacy/DiplomacyManager.kt

```diff
@@ -784,8 +784,6 @@ class DiplomacyManager() : IsPartOfGameInfoSerialization {
             }
             civInfo.addNotification("We have canceled our Defensive Pact with [${it.otherCivName}]!",
                 NotificationCategory.Diplomacy, NotificationIcon.Diplomacy, it.otherCivName)
-            it.otherCiv().addNotification("[${civInfo.civName}] has canceled our Defensive Pact with us!",
-                NotificationCategory.Diplomacy, civInfo.civName, NotificationIcon.Diplomacy)
```

Comment:
I decided to drop the "us" notification because it would require more translation and complexity. I would need to filter it out of the getCommonCivs(), and it's inconsistent because we don't use it for the other notifications.

Side note: Does anyone care that signing a DoF sends two notifications to each Civ? This tends to clutter the screen a bit.